### ISSUE TITLE
Add RF system block diagram components

### ISSUE THREAD

**Issue by rossmeier** [fenjalien/circuitypst#6]
Hi,

thanks for working on this. Circuitikz includes a bunch of features about (mostly RF) system diagram. Do you consider them in scope for this project?
I have started working on some components and utility functions to use them and would be glad to contribute them here if that is desired
![image](https://github.com/fenjalien/cirCeTZ/assets/4528595/7599be4a-9f45-47ea-9d85-e7a95bf37b6e)

**Comment by mibmo** [comment]
@fenjalien any update on this? :)

**Comment by fenjalien** [comment]
Yes all circuit components will be considered. Bear in mind that there is a rework on the dev branch that is slowly being worked on.

**Comment by rossmeier** [comment]
I put my code here: https://github.com/rossmeier/cetz-rf
If somebody wants to use it before it (or another implementation) finds its way into ciCeTZ.
